about.html

```diff
@@ -0,0 +1,148 @@
+                        </p>
+                    </div>
+
+                    <p class="mt-6 text-center bg-white font-semibold p-2">
+                        contact us via email if you need any help along the way <br>
+                        <a class="text-primary no-underline" href="mailto:info@pythonpulse.org">
+                            pythonpulse.org
+                        </a>
+                    </p>
+
+                    <div class="mt-6 flex items-center justify-center">
+                        <img class="h-20 inline" src="assets/pplogo.png">
+                        <span class="text-primary text-5xl font-bold">Python Pulse</span>
+                    </div>
+
+                    <div>
+                        <h3 class="font-bold text-4xl text-secondary mt-10 text-center" >
+                            See past events
+                        </h3>
+                        <p class="text-center">
+                            Check out our past events to see what we've been up to
+                        </p>
+                        
+                        <div class="grid-rows-4">
+                            <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2 gap-4 px-4 mt-10">
+                                
+                                <div class="bg-orange-800/80 hover:bg-orange-800/0 transition-all ease-linear duration-500 bg-cover bg-blend-overlay bg-center min-h-[250px] conference1">
+                                    <div class="w-full h-full flex flex-col items-center justify-center">
+                                        <h4 class="text-white text-2xl font-bold" > 2020</h4>
+                                        <p class="text-white mt-4 text-center">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.</p>
+
+                                    </div>
+                                </div>
+
+                                <div class="bg-orange-800/80 hover:bg-orange-800/0 transition-all ease-linear duration-500 bg-cover bg-blend-overlay bg-center min-h-[250px] conference2">
+                                    <div class="w-full h-full flex flex-col items-center justify-center">
+                                        <h4 class="text-white text-2xl font-bold" > 2021</h4>
+                                        <p class="text-white mt-4 text-center">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.</p>
+
+                                    </div>
+                                </div>
+                                
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see from [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in the about page on the mobile screen we have both the `partners` section and `footer` section so please make sure that you work on that.

| Your Design |Cindy Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/225571419-03d49737-168a-4699-8aa8-25af3beaecc3.png)| ![image](https://user-images.githubusercontent.com/92303891/225571662-009d3fff-0f14-4258-836f-5ece96e3b4c8.png) |